nmdc_runtime/dagster.Dockerfile

```diff
@@ -16,15 +16,18 @@ RUN export DEBIAN_FRONTEND=noninteractive && \
   apt-get -y clean && \
   rm -rf /var/lib/apt/lists/*
 
+# Install Poetry.
+RUN pip install poetry
```

Comment:
what version of poetry? is there a "go-to" docker image that distributes poetry?